docs/userguide/usingdocker.md

```diff
@@ -43,18 +43,18 @@ version information on the currently installed Docker client and daemon.
 This command will not only provide you the version of Docker client and
 daemon you are using, but also the version of Go (the programming
 language powering Docker).
-
-    Client version: 0.8.0
-    Go version (client): go1.2
-
-    Git commit (client): cc3a8c8
-    Server version: 0.8.0
-
-    Git commit (server): cc3a8c8
-    Go version (server): go1.2
-
-    Last stable version: 0.8.0
-
+  
+    Client version: 1.7.1
```

Comment:
@charleswhchan Thank you for the contribution.  I noticed the format of this command has changed.

```
Client:
 Version:      1.8.0-dev
 API version:  1.21
 Go version:   go1.4.2
 Git commit:   0f49141
 Built:        Tue Aug  4 17:42:34 UTC 2015
 OS/Arch:      linux/amd64

Server:
 Version:      1.8.0-dev
 API version:  1.21
 Go version:   go1.4.2
 Git commit:   0f49141
 Built:        Tue Aug  4 17:42:34 UTC 2015
 OS/Arch:      linux/amd64
```